data/brands/amenity/bank.json

```
@@ -11064,6 +11064,18 @@
         "name:en": "ProCredit Bank"
       }
     },
+    {
+      "displayName": "ПроКредит Банк (Україна)",
+      "locationSet": {"include": ["ua"]},
+      "tags": {
+        "amenity": "bank",
+        "brand": "ПроКредит Банк",
+        "brand:en": "ProCredit Bank",
+        "brand:wikidata": "Q20091911",
+        "name": "ПроКредит Банк",
+        "name:en": "ProCredit Bank"
```

```suggestion
        "name:en": "ProCredit Bank",
        "name:uk": "ПроКредит Банк"
```

What about add name:uk/brand:uk, the same as name/brand? See https://wiki.openstreetmap.org/wiki/Multilingual_names#Repeating_name_with_language_specific_tag